criu/page-pipe.c

```diff
@@ -120,7 +121,13 @@ static struct page_pipe_buf *ppb_alloc(struct page_pipe *pp, unsigned int ppb_fl
 		cnt_add(CNT_PAGE_PIPES, 1);
 
 		ppb->pipe_off = 0;
-		ppb->pipe_size = fcntl(ppb->p[0], F_GETPIPE_SZ, 0) / PAGE_SIZE;
```

Comment:
Just wonder if it's a real case when `fcntl(ppb->p[0], F_GETPIPE_SZ, 0)` fails? As far as I can see from the kernel code it can only fail if `ppb->p[0]` is not a pipe FD, but `ppb->p[0]` is always a pipe FD because it is created a few lines above.

Reply:
Thank you for your reply.
From the perspective of using security, it should be necessary to increase the judgment, otherwise the failure will not be recognized.